package.json

```diff
@@ -40,23 +40,34 @@
     "cli-progress": "^3.8.2",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
+    "crc": "^3.8.0",
     "encryptedfs": "^0.1.0",
     "fs-extra": "^9.0.0",
     "futoin-hkdf": "^1.3.2",
+    "git-list-pack": "0.0.10",
+    "http-parser-js": "^0.5.1",
     "inquirer": "^7.1.0",
     "inquirer-fuzzy-path": "^2.3.0",
-    "isomorphic-git": "^0.35.1",
+    "isomorphic-git": "^1.5.0",
     "jsonfile": "^6.0.1",
     "kbpgp": "^2.0.82",
+    "marky": "^1.2.1",
```

Comment:
This looks like a dev dependency?

Reply:
I originally thought this was for providing information to the progress hook on the client side but in any case, I've removed it, if we want progress shown on the pulling side, we could probably come up with a better way to do progress

Reply:
I've taken it out for now

Reply:
This is actually something I reckon we can build ourselves (at least from a timing perspective). And I also think the visual aspect of progress bar componentry would probably be in Polykey repo and not here.